src/components/related/OutfitList.jsx

```diff
@@ -1,14 +1,56 @@
 import React, { useState, useEffect } from 'react';
 import styles from './related.module.css';
-import OutfitCard from './OutfitCard.jsx'
+import axios from 'axios';
+import {useProductContext} from '../../context/ProductContext.jsx';
+import OutfitCard from './OutfitCard.jsx';
+import plusSymbol from '../../../asset/img/plus-symbol.png';
+
 
 const OutfitList = () => {
 
+  const { info } = useProductContext();
+  const [outfitData, setOutfitData] = useState([]);
+
+  //get stored outfit list
+  const getStoredOutfit = () =>{
+    axios.get('/outfit')
+    .then(results => {
+      setOutfitData(results.data)
+    })
+  }
+
+  useEffect(() =>{
+    getStoredOutfit()
+  },[])
+
+  // post data to server, then get the new outfit list back and reset outfitData
+  const handleAddOutfit = () => {
+    axios.post('/outfit', {
+      productID: info.id,
+      productInfo: info
+    })
+    .catch(function (err) {
+      console.log('error in handleAddOutfit', err);
+    });
+
+    getStoredOutfit()
```

Comment:
Seems that getStoredOutFite() function call is grabbing the new outfit list that included the outfit that is posted to the server. Would it be safer to call this function after the Promise from the axios post request? Possibly add a 'then' that would include the getStoredOutfit()  in line 31. That way, after the axios post Promise is resolved the it will call the function and grab the list with the new out fit.

Reply:
I put the getStoredOutfit() in .then, but nothing returns back. It took me a while to figure it out where to place getStoredOutfit(). It's still looks counterintuitive to me

Reply:
Got it. Maybe its something we can look into when we start refactoring. But if its functioning as expected and not breaking anything we should be good.